mesheryctl/cmd/status.go

```diff
@@ -28,12 +27,12 @@ var statusCmd = &cobra.Command{
 	Short: "Check Meshery status",
 	Long:  `Check status of Meshery and Meshery adapters.`,
 	Run: func(cmd *cobra.Command, args []string) {
-		fmt.Println("Meshery containers status . . .\n")
+		log.Info("Meshery containers status . . .\n")
 		out, err := exec.Command("docker-compose", "-f", dockerComposeFile, "ps").Output()
 		if err != nil {
 			log.Fatal("[ERROR] Please, install docker-compose. The error message: \n", err)
```

Comment:
I think it will good to follow a pattern, whether or not to include [ERROR] in fatal msgs. . . lets take it out.